# HG changeset patch
# Parent  a0c1768fb594da51951048f6b167ab5a9499d734
Ignore my packet captures directories

I never found myself looking for an older pcap, what we really need is a bulb
simulator and test scenarios.

diff --git a/.hgignore b/.hgignore
--- a/.hgignore
+++ b/.hgignore
@@ -1,3 +1,4 @@
 .*\.sw[a-z]$
 .*\.py[co]$
 ^build
+^pcaps
